java/acorn-comparison/src/main/java/com/datasqrl/ai/comparison/MicrometerObservability.java

```diff
@@ -62,6 +62,14 @@ public ModelInvocation start() {
     return new StandardModelInvocation();
   }
 
+  @Override
+  public int timeoutBetweenRequestsSeconds(String modelProvider) {
+    return switch (modelProvider) {
+      case "groq" -> 30;
+      default -> 0;
+    };
+  }
+
```

Comment:
@mbroecheler What do you think of this? Should we build this into the configuration instead? It's really only needed when we automate many calls to the models